_docs/collection-details.md

```diff
@@ -29,25 +30,24 @@ To initiate a GHS mobile collections please use the following details:
 
 {% include language-tabbar.html prefix="collection-ghs" raw=data-raw %}
 
-Once the transaction is created the specified phone number will receive instructions on how to pay in the requested amount.
+Once the transaction is created, instructions for completing payment will be sent to the specified phone number.
 
-Please note that for MTN Cash,if the customer does not have enough funds to pay the transaction they might not receive the payment prompt to their phone. In this case the transaction request should be re-sent, once the customer has confirmed they have enough funds.
-
-Once the payment has been successfully done a `transaction.paid_in` webhook will be sent out.
+Once the funds have been successfully received from the sender, `payin_method.paid_in` and `transaction.paid_in` webhooks will be sent out.
```

Comment:
Can we get some pictures here? We had plenty of tests, so we should be able to sanitize some and put some nice pictures on how the flow actually works

Reply:
Agree this would be great, my only concern is that currently the screenshots leak the provider info as the receiver. If this is customisable at the provider end I suggest we hold off until we have screenshots which indicate the payment is going to AZA rather than the provider. Likewise I don't think it's a huge problem, so we can add current ones temporarily.

Reply:
Any provider we can mosaic the provider stuff out easily?